Tree.h

```diff
@@ -132,13 +132,21 @@ class LeafNode : public TreeNode<T> {
 
 // load a regression tree from Json
 template <class T>
-TreeNode<T>* fromJson(const folly::dynamic& obj) {
-  int index = obj["index"].asInt();
+TreeNode<T>* fromJson(const folly::dynamic& obj, const Config& cfg) {
+  const folly::dynamic* feature = nullptr;
+  try {
+    feature = &obj["feature"];
+  } catch(...) {
+  }
```

Comment:
Are you suggesting use **find**? I tried **find** but got the following compilation error:

/usr/local/include/folly/dynamic-inl.h:664:32: error: ‘get’ is not a member of ‘folly::dynamic::GetAddrImpl<folly::dynamic::const_item_iterator>’
   return GetAddrImpl<T>::get(u_);
                                ^